packages/textarea/src/component.tsx

```diff
@@ -41,25 +42,25 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
 
     return (
       <div
-        className={classNames(className, {
-          'input mb-0': true,
-          'input--is-invalid': isInvalid,
-          'input--is-disabled': disabled,
-          'input--is-read-only': readOnly,
-        })}
         style={style}
       >
         {label && (
           <label htmlFor={id}>
             {label}
             {optional && (
-              <span className="pl-8 font-normal text-14 text-gray-500">
+              <span className={input.optional}>
                 (valgfritt)
               </span>
             )}
           </label>
         )}
         <textarea
+          className={classNames({
```

Comment:
In the previous version `className` prop was also included. Was it intentionally removed?

Reply:
Which version 😊 I can add `className` prop. Do we need it? Like that you mean? 👇 
```suggestion
          className={classNames(props.className,{
```

Reply:
Yes, that's what was there before as I see in your changed files (by previous version I meant `master` version). Not sure if that's still needed but I thought we should be careful about removing stuff like that in case we break something for users that need to migrate from Fabric to Warp. Another thing is that in `master` the `className` prop was applied to the wrapping `div`, not the `textarea` element. Whether that makes things behave differently - I'm again not sure.

Reply:
Ah do you meant here https://github.com/warp-ds/react/pull/16/files#diff-e6dfac355d15645c03e0c5dea277d37e1d18bb5460789f49189f9c8ab4b130fdL44?
I've "refurnitured" a bit here as for example this one https://github.com/fabric-ds/css/blob/49b07c05686ef132b203026e941bf04dbab6fc7a/src/components/form-elements.css#L42 has `& input` inside the class and that is not what we have now, right? As we want to have classes for specific elements. I've tested and the look and feel is preserved but maybe I'm missing something

Reply:
Moving the invalid/disabled/readonly classes down to respective elements makes sense, yes. What I'm not sure about is whether we should keep `className` prop on the textarea element or rather the wrapping div like it was before. So something like this:
```
<div
   className={className}
   style={style}
>
```

Does that make sense?

Reply:
Ok, I see. Let's keep it on the parent. Good catch 👍

Reply:
Fixed https://github.com/warp-ds/react/pull/16/files#diff-e6dfac355d15645c03e0c5dea277d37e1d18bb5460789f49189f9c8ab4b130fdR45